transcom/mymove
# INT-B-21042

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1007456&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

Allows Prime to create shipments with second and third pickup/destination addresses.

## Testing
1. Login as Prime
2. Open existing move
3. Click Create Shipment
4. Enter 2nd/3rd addresses
5. Save
6. Check DB for 3rd addresses
